README.md

```diff
@@ -199,6 +184,4 @@ I would like to thank microverse.org for giving me this oportunity.
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
To make the repo more professional, kindly link to your license file by changing `./LICENSE` to `./MIT.md` because the link is redirecting to a 404 page.